src/components/Pages/StoriesPage/StoryFormButtonModal.js

```diff
@@ -63,7 +71,7 @@ class StoryFormButtonModal extends Component {
             }}
           >
             <Toast.Body className={"Success"}>
-              <h6>Your testimonial has been submitted! </h6>
+              <h6>Submission was successful </h6>
```

Comment:
Please not change the text for existing code.   Pass in submission type as a prop, and substitute it here "testimonial" "event" "action" "service provider"

Reply:
Please see latest commit